build.gradle

```diff
@@ -58,6 +66,11 @@ sourceSets {
   }
 }
 
+jar {
+  // Include .java files for GWT compatibility.
+  from sourceSets.main.allJava
```

Comment:
Is this typical for gwt-compatible projects? It seems wrong to include the sources in the binary jar, instead the gwt compiler should put the sources jar on its classpath.

Reply:
Sorry - what I actually meant to do in this PR was to add the xml+supersources into the sources jar, and leave the binary jar unchanged. Fixed.